.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -36,8 +36,8 @@ assignees: ''
 * OpenCAS version (commit hash or tag):
 * Operating System:
 * Kernel version:
-* Cache device type (model):
-* Core device type (model):
+* Cache device type (SSD/Optane/NAND/other):
```

Comment:
SSD ⊃ {NAND, Optane}, I think SSD is not needed here

Reply:
okey-dokey

Reply:
what about Sata SSD?